### PR TITLE
Adapt create service texts for quality gates only

### LINKED ISSUE #4172: Show `keptn create service` when Bridge is used for Quality-Gates only
### User story

As a user, I use Keptn only for quality gates, but some hints are leading me to the instructions of `keptn onboard service`. This command does not make sense, since I'm not deploying my app with Keptn. 

image

#### Details
- Identify all spots where the `keptn onboard service` cmd is used.
- To control the mode in which the Bridge is used, the Bridge knows the environment variable: `KEPTN_INSTALLATION_TYPE` = `QUALITY_GATES,CONTINUOUS_OPERATIONS,CONTINUOUS_DELIVERY`
- Just in case of `CONTINUOUS_DELIVERY` the `keptn onboard service` hints should be displayed.
- Going forward the `keptn onboard` command will be replaced by `keptn create service` anyways. 

#### Acceptance criteria
- [x] When using the Bridge for quality gates only, I get the hint to use `keptn create service`

### PR DESCRIPTION
**Pull request by laneli**
Closes #4172 

image

image

image

Signed-off-by: Elisabeth Lang <elisabeth.lang@dynatrace.com>

### CODE REVIEW

**Comment by Kirdock** [comment]
Those messages should be the same on each component.
For the quality-gates-only use case, we have `No service created yet. Follow the instructions to create a service` and `No service onboarded yet. Follow instructions to create a service`

File: bridge/client/app/_views/ktb-service-view/ktb-service-view.component.html [hunk 1]
```diff
@@ -8,9 +8,18 @@
...
-        <p>No service onboarded yet. Follow the instructions to <a [href]="'/manage/service/#onboard-a-service' | keptnUrl" target="_blank" rel="noopener noreferrer">onboard a service</a>.</p>
+      <div fxLayout="row" fxLayoutAlign="start" class="mt-2" *ngIf="project.getServices().length == 0" uitestid="keptn-service-view-noServices">
+        <dt-icon fxFlex="20px" class="icon" name="information"></dt-icon>
+        <p class="m-0 ml-2">
+          <ng-container *ngIf="!isQualityGatesOnly">
+            No service onboarded yet. Follow the instructions to
+            <a [href]="'/manage/service/#onboard-a-service' | keptnUrl" target="_blank" rel="noopener noreferrer">onboard a service</a>.
+          </ng-container>
+          <ng-container *ngIf="isQualityGatesOnly">
+            No service created yet.
```
Comment (Kirdock) [inline bridge/client/app/_views/ktb-service-view/ktb-service-view.component.html:19]: `Follow the instructions to` is missing here

Comment (laneli) [reply bridge/client/app/_views/ktb-service-view/ktb-service-view.component.html]: Done

File: bridge/client/app/_components/ktb-services-list/ktb-services-list.component.html [hunk 2]
```diff
@@ -56,9 +56,17 @@
...
+  <div fxLayout="row" fxLayoutAlign="start" uitestid="keptn-services-list-noServices">
+    <dt-icon fxFlex="20px" class="icon" name="information"></dt-icon>
+    <p class="m-0 ml-2">
+      <ng-container *ngIf="!isQualityGatesOnly">
+        No service onboarded yet.<br/>Follow the instructions to
+        <a [href]="'/manage/service/#onboard-a-service' | keptnUrl" target="_blank" rel="noopener noreferrer">onboard a service</a>.
+      </ng-container>
+      <ng-container *ngIf="isQualityGatesOnly">
+        No service created yet.<br/>Follow the instructions to
+        <a *ngIf="isQualityGatesOnly" [href]="'/manage/service/#create-a-service' | keptnUrl" target="_blank" rel="noopener noreferrer">create a service</a>.
```
Comment (Kirdock) [inline bridge/client/app/_components/ktb-services-list/ktb-services-list.component.html:68]: `*ngIf="isQualityGatesOnly"` is not necessary here

Comment (laneli) [reply bridge/client/app/_components/ktb-services-list/ktb-services-list.component.html]: Resolved

File: bridge/client/app/_components/ktb-services-list/ktb-services-list.component.ts [hunk 3]
```diff
@@ -36,6 +35,7 @@ export class KtbServicesListComponent implements OnInit, OnDestroy {
   public _services: Service[] = [];
   public _pageSize: number = DEFAULT_PAGE_SIZE;
   public dataSource: DtTableDataSource<Service>;
+  public isQualityGatesOnly: boolean;
```
Comment (Kirdock) [inline bridge/client/app/_components/ktb-services-list/ktb-services-list.component.ts:38]: Unused variable

**Review by Kirdock — APPROVED** [review]

**Pull request merged**